bazel/linux/templates/kunit.template.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+echo ========= {message} - {target} ==========
+
+function load {
+	echo "... loading $@."
+	insmod "$@"
+}
+
+set -e
+{commands}
```

Comment:
without the blob I had before with `cd ...` it doesn't work when a rootfs is specified explicitly. Can you add it back?

(note to self: we should really start adding unit tests for those rules, it's to the point where it is too easy to regress)

Reply:
Added the blob back.